tslint.json

```diff
@@ -1,40 +1,26 @@
 {
+  "defaultSeverity": "error",
+  "extends": ["tslint:all", "tslint-config-prettier"],
   "rules": {
-    "file-name-casing": [true, "camel-case"],
-    "class-name": true,
-    "comment-format": [true, "check-space"],
-    "indent": [true, "spaces", 2],
-    "one-export": false,
-    "no-console": true,
-    "no-debugger": true,
-    "no-eval": true,
-    "no-internal-module": true,
-    "no-trailing-whitespace": true,
-    "no-unsafe-finally": true,
-    "no-var-keyword": true,
-    "one-line": [true, "check-open-brace", "check-whitespace"],
-    "quotemark": [true, "single", "avoid-escape"],
-    "semicolon": [true, "never"],
-    "triple-equals": [true, "allow-null-check"],
-    "typedef-whitespace": [
+    "completed-docs": [
       true,
       {
-        "call-signature": "nospace",
-        "index-signature": "nospace",
-        "parameter": "nospace",
-        "property-declaration": "nospace",
-        "variable-declaration": "nospace"
+        "interfaces": {
+          "visibilities": ["exported"]
+        },
+        "classes": {
+          "visibilities": ["exported"]
+        }
       }
     ],
-    "variable-name": [true, "ban-keywords"],
-    "whitespace": [
-      true,
-      "check-branch",
-      "check-decl",
-      "check-operator",
-      "check-separator",
-      "check-type"
-    ],
-    "ordered-imports": true
+    "variable-name": [true, "allow-pascal-case"],
+    "typedef": [true, "call-signature"],
+    "file-name-casing": [true, "camel-case"],
+    "no-implicit-dependencies": [false, "dev"],
+    "only-arrow-functions": false,
+    "no-magic-numbers": false,
+    "no-parameter-properties": false,
+    "no-unused-expression": true,
```

Comment:
not working for some reason, there are still unused imports all over the place and `tsc` doesn't complain. And `no-unused-variable` is deprecated, and is throwing weird errors.